ffmpeg/logging.h

```diff
@@ -1,6 +1,13 @@
 #ifndef _LPMS_LOGGING_H_
 #define _LPMS_LOGGING_H_
 
+//MACRO FOR DEVELOP
+#define USE_LVPDNN_
+#ifdef USE_LVPDNN_
```

Comment:
Is the main purpose of this macro to hardcode the variables below during development? And this macro wouldn't be used in production?

Reply:
Bump